pyspecdata/core.py

```diff
@@ -6850,15 +6862,16 @@ def myfilter(x,center = 250e3,sigma = 100e3):
 
 #{{{ fitdata
 class fitdata(nddata):
+    r''' Inherits from an nddata and enables curve fitting through use of a sympy expression.
+    '''
```

Comment:
this is where I would add a little bit of documentaiton

Reply:
Yes, agreed

Reply:
I can't see any changes yet -- definitely feel free to make very incremental commits and push them up, because then I can go through these on the fly

Reply:
Yes - I am pushing them now as I do them